include/oneapi/dnnl/dnnl_graph_types.h

```diff
@@ -201,6 +201,8 @@ typedef enum {
     dnnl_graph_op_exp,
     dnnl_graph_op_gelu,
     dnnl_graph_op_gelu_backward,
+    dnnl_graph_op_gen_index,
+    dnnl_graph_op_greater_equal,
```

Comment:
Add them at the end of the list, otherwise it breaks ABI.

Reply:
thanks for the reminder, updated.